Makefile

```diff
@@ -1,81 +1,9 @@
-CLIENT_BINARY_NAME = ktk
```

Comment:
All the contents of this Makefile seemed to be one for a generated project.
So, I've moved it to `example/scaffold/Makefile` accordingly.